packages/quantic/force-app/main/default/lwc/quanticCaseAssistInterface/quanticCaseAssistInterface.js

```diff
@@ -0,0 +1,75 @@
+import {LightningElement, api} from 'lwc';
+import {
+  getHeadlessBindings,
+  loadDependencies,
+  setEngineOptions,
+  setInitializedCallback
+} from 'c/quanticHeadlessLoader';
+// @ts-ignore
+import getHeadlessConfiguration from '@salesforce/apex/HeadlessController.getHeadlessConfiguration';
+
+/** @typedef {import("coveo").CaseAssistEngine} CaseAssistEngine */
+/** @typedef {import("coveo").CaseAssistEngineOptions} CaseAssistEngineOptions */
+
+/**
+ * The `QuanticCaseAssistInterface` component handles the headless case assist engine.
+ * A single instance should be used for each instance of the Coveo Headless case assist engine.
+ *
+ * @example
+ * <c-quantic-case-assist-interface engine-id={engineId} case-assist-id={caseAssistId}></c-quantic-case-assist-interface>
+ */
+export default class QuanticSearchInterface extends LightningElement {
+  /**
+   * The ID of the engine instance the component registers to.
+   * @api
+   * @type {string}
+   */
+  @api engineId;
+
+  /**
+   * The Case Assist configuration ID.
+   * @api
+   * @type {string}
+   */
+  @api caseAssistId;
+
+  /** @type {CaseAssistEngineOptions} */
+  engineOptions;
+
+  connectedCallback() {
+    loadDependencies(this, 'case-assist').then(() => {
```

Comment:
instead of using the string here we could export the `HeadlessBundleNames` object from quanticHeadlessLoader and use it here to make sure the strings always match.

Reply:
If you want to do that then we should do the same in the `QuanticSearchInterface` as well.

Reply:
Do I include this in this PR ?

Reply:
yes

Reply:
In the `QuanticSearchInterface`, loadDependencies is not taking any headlessUseCase parameter so no strings there. It's taking by default the correct Headless bundle name, so I  guess it fine right?

Reply:
Yep sounds correct. Thanks for looking into it!